app/js/controllers/relatedCtrl.js

```diff
@@ -1,13 +1,17 @@
 KMCMenu.controller('relatedCtrl', ['$scope',
 	function ($scope) {
 		$scope.relatedOption = "relatedToEntry";
-		$scope.entryList = "";
+		$scope.entryList = [];
 		$scope.playlistId = {id:'' ,text:''};
+		if(!("related" in $scope.playerData.config.plugins)) {
+			$scope.playerData.config.plugins["related"] = {disable: true};
+			$scope.playerData.config.plugins["related"].useContext = true;
+		}
 
 		// init radio buttons according to selected related entries source
-		if ($scope.playerData.config.player.plugins && $scope.playerData.config.player.plugins["related"]) {
-			var data = $scope.playerData.config.player.plugins["related"];
-			if (data.entryList && data.entryList !== ""){
+		if ($scope.playerData.config.plugins["related"].enabled) {
+			var data = $scope.playerData.config.plugins["related"];
+			if (data.entryList?.length > 0){
```

Comment:
```suggestion
			if (data.entryList?.length){
```

Reply:
is this code transpiled ? because optional chaining isn't supported by some browsers

Reply:
i think is less readable

Reply:
is the code transpiled ?